cmd/protoc-gen-openapi/generator/openapi-v3.go

```diff
@@ -127,6 +135,10 @@ func (g *OpenAPIv3Generator) buildDocumentV3() *v3.Document {
 		g.requiredSchemas = g.requiredSchemas[count:len(g.requiredSchemas)]
 	}
 
+	for _, schema := range g.modifiedSchemas {
+		g.addSchemaToDocumentV3(d, schema.Message, schema.Name, schema.FieldNames)
```

Comment:
not sure what best practice is here, but you can pass the entire `schema` here since you seem to need all of its attributes